### PR TITLE
Add benchmark info and prereqs in EKS install plus misc GKE edits

### PR DESCRIPTION
**Pull request by johnsmartco**
**What this PR does**:

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Checklist**
- [ ] Changes manually tested
- [ ] Automated Tests added/updated
- [ ] Documentation added/updated
- [ ] CHANGELOG.md updated (not required for documentation PRs)
- [ ] CLA Signed:  [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: docs/content/en/install/eks/_index.md [hunk 1]
```diff
@@ -40,24 +40,138 @@ On this infrastructure the K8ssandra installation will consist of the following
...
+* The performance of K8ssandra running on Amazon EKS, Google GCP GKE, and Microsoft Azure AKS. 
+
+It's important to note the following additional AWS infrastructure settings used in the benchmark:
+
+* 8 to 16 vCPUs 
+  * r5 instances: Intel Xeon Platinum 8000 series. Cassandra workloads are mostly CPU bound and the core speed made a difference in the throughput benchmarks.
+* 32 GB to 128 GB RAM (we used 64 GB RAM during the benchmark)
+* 2 to 4 TB of disk space
+  * In the benchmark, we used 1x 3.4 TB EBS gp2 volume
+* 10k Input/Output Operations per Second (IOPS)
```
Comment (bradfordcp) [inline docs/content/en/install/eks/_index.md:59]: Did we specify IOPS or were these just observed values?

File: docs/content/en/install/eks/_index.md [hunk 2]
```diff
@@ -40,24 +40,138 @@ On this infrastructure the K8ssandra installation will consist of the following
...
+
+For the disk performance, the benchmark used [Cassandra inspired fio profiles](https://github.com/ibspoof/cassandra-fio) that attempt to emulate Leveled Compaction Strategy and Size Tiered Compaction Strategy behaviors.  
+
+Regarding the Cassandra version and settings:
+
+* The benchmark used Cassandra 4.0-beta4.
+* Cassandra default settings were applied with the exception of garbage collection (GC) settings. This used G1GC with 31GB of heap size, along with a few GC related JVM flags:
+
+  ``` bash
```
Comment (bradfordcp) [inline docs/content/en/install/eks/_index.md:68]: I would remove `bash` here as it is not a bash script.

File: docs/content/en/install/eks/_index.md [hunk 3]
```diff
@@ -40,24 +40,138 @@ On this infrastructure the K8ssandra installation will consist of the following
...
+curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 > get_helm.sh
+chmod 700 get_helm.sh
+./get_helm.sh
+```
+
+#### Install Python v3
+
+If you haven't already, install Python version 3.x for your OS. See the [python downloads](https://www.python.org/downloads/) page.
+
+
```
Comment (bradfordcp) [inline docs/content/en/install/eks/_index.md:174]: Do we really need to cover installation of the various tools? That's the whole purpose of the links in the table.

**Review by emerkle826 — APPROVED** [review]
LGTM :+1:

File: docs/content/en/install/eks/_index.md (hunk 3 above)
Comment (bradfordcp) [reply docs/content/en/install/eks/_index.md:174]: Follow-up, are we going to cover installation of tools in the other topics as well?

Comment (johnsmartco) [reply docs/content/en/install/eks/_index.md:174]: My recommendation is we briefly cover these prereq install steps for newbie users along with links to related cloud docs. Yes, doing the same in our cloud install topics.

File: docs/content/en/install/eks/_index.md (hunk 1 above)
Comment (johnsmartco) [reply docs/content/en/install/eks/_index.md:59]: Right, I think the 10k IOPS were just observed values in Alex's benchmark report, https://k8ssandra.io/blog/articles/k8ssandra-performance-benchmarks-on-cloud-managed-kubernetes/.

File: docs/content/en/install/eks/_index.md [hunk 4]
```diff
@@ -389,7 +389,33 @@ Take note of the comments in this file. If you have changed the name of your sec
 
 ### Deploy K8ssandra with Helm
 
-With a `values.yaml` file generated which details out specific configuration overrides we can now deploy K8ssandra via Helm.
+If you haven't already, add the latest stable K8ssandra repo:
+
+```bash
+helm repo add k8ssandra https://helm.k8ssandra.io/stable
```
Comment (bradfordcp) [inline docs/content/en/install/eks/_index.md:395]: You can drop the `stable`

File: docs/content/en/install/eks/_index.md (hunk 1 above)
Comment (johnsmartco) [reply docs/content/en/install/eks/_index.md]: Updated in more recent commit.

File: docs/content/en/install/eks/_index.md (hunk 2 above)
Comment (johnsmartco) [reply docs/content/en/install/eks/_index.md]: Updated in recent commit.

File: docs/content/en/install/eks/_index.md (hunk 4 above)
Comment (johnsmartco) [reply docs/content/en/install/eks/_index.md]: Thanks Chris - updated in recent commit.

**Pull request merged**